art_show/templates/art_show_reports/index.html

```diff
@@ -13,17 +13,22 @@
     <h3 class="panel-title">Art Show Reports</h3>
   </div>
   <div class="panel-body">
+    <p>
     <a class="btn btn-default" href="sales_invoices">Sales Invoices</a>
-    <a class="btn btn-default" href="high_bids">High Bids Door Report</a>
+    <a class="btn btn-primary" href="artist_invoices">Artist Payout Invoices</a>
```

Comment:
Cheating slightly here -- since I moved this link, I expect we'll have a merge conflict with https://github.com/MidwestFurryFandom/art_show/pull/213, so I added the page link here so I can just accept this branch's version of the file.